Dockerfile

```diff
@@ -7,9 +8,14 @@ RUN apk --update add git
 
 RUN npm install
 
-RUN adduser -u 9000 -D app
-USER app
+
+RUN apk del --purge git
```

Comment:
If the intention here is to keep image size down, this won't do it since it's in a separate layer. Something like:

```
RUN apk --update add git && npm install && apk del --purge git
```

Would do that though. If the intent is just to keep git out of the final image, then it doesn' t matter.